lighthouse-core/test/audits/network-requests-test.js

```diff
@@ -6,28 +6,31 @@
 'use strict';
 
 const NetworkRequests = require('../../audits/network-requests.js');
-const assert = require('assert');
 const networkRecordsToDevtoolsLog = require('../network-records-to-devtools-log.js');
 
 const acceptableDevToolsLog = require('../fixtures/traces/progressive-app-m60.devtools.log.json');
+const cutoffLoadDevtoolsLog = require('../fixtures/traces/cutoff-load-m83.devtoolslog.json');
```

Comment:
introduced a new test devtoolsLog/trace of a load of the report viewer, devtools throttling, cut off at 1500ms (via `--max-wait-for-load`). Six network requests, three unfinished.

Seems handy to have a kind of gross load like that with relatively small file size.